README.old.md

```diff
@@ -0,0 +1 @@
+# space-travelers-hub
```

Comment:
Kindly delete this README file as it is not useful and shouldn't be in your project directory 👍